packages/web-api/src/methods.ts

```diff
@@ -2170,6 +2172,14 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
     messages: bindApiCall<SearchMessagesArguments, SearchMessagesResponse>(this, 'search.messages'),
   };
 
+  public readonly slacklists = {
+    /**
+     * @description Create a List.
+     * @see {@link https://docs.slack.dev/reference/methods/slacklists.create slackLists.create` API reference
```

Comment:
```suggestion
     * @see {@link https://docs.slack.dev/reference/methods/slacklists.create `slackLists.create` API reference}.
```

📚 quibble: Formatting for `jsdoc` previews!